src/auth/strategies/google.strategy.ts

```diff
@@ -0,0 +1,36 @@
+import { PassportStrategy } from "@nestjs/passport";
+import { Strategy, VerifyCallback } from "passport-google-oauth20";
+import { config } from "dotenv";
```

Comment:
We should use our own `configService`

Reply:
Will do, saw it but was confused so just went for the typical approach 😄